src/charm.py

```diff
@@ -38,22 +33,22 @@ def __init__(self, *args: typing.Any) -> None:
             args: passthrough to CharmBase.
         """
         super().__init__(*args)
-        self._databases = Databases(charm=self)
         self._secret_storage = SecretStorage(charm=self)
-
         try:
             self._charm_state = CharmState.from_charm(
                 charm=self, secret_storage=self._secret_storage
             )
         except CharmConfigInvalidError as exc:
             self._update_app_and_unit_status(ops.BlockedStatus(exc.msg))
             return
-        self._flask_app = FlaskApp(charm_state=self._charm_state)
         self._webserver = GunicornWebserver(
```

Comment:
This should also not be a class, right?

Reply:
Since the `webserver` doesn't cause any trouble, the modification of that class is not in high priority, we can do that later, thanks!